string name                         # object's name
bool allow                          # true - allow / false - deny
--- 
